code-challenges.test.js

```diff
@@ -50,3 +122,30 @@ const cubeAndSum2 = [0, 5, 10]
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+
+//const cubeAndSum1 = [2, 3, 4]
+let cubed = 0
+
+for(let i = 0; 1 < cubeAndSum1.length; i++) {
+    cubed += cubeAndSum1[i]
+}
+console.log(cubeAndSum1)
+
+//same for 
+//const cubeAndSum2 = [0,5,10]
+let cubed2 = 0
+
+for(let i = 0; 1 < cubeAndSum2.length; i++) {
+    cubed2 += cubeAndSum2[i]
+}
+console.log(cubeAndSum2)
+
+//relating it back to the describe statement?
+//numbersCubed
+// I'm thinking it would be something like,
+// const numbersCubed = cubeAndSum1 ?
+// const numbersCubed = cubeAndSum2 ? 
+// My terminal stopped working, I cannot run any tests
+
+
+//(I've attached a practice.js file that explains my logic more.) Sometimes the tests running confuses me and I have to work out of a separate file.
```

Comment:
I looked at the practice.js file. The main issue in all of these challenges is that you're never actually making a function. So, when the jest test performs the function call in the expect statement, it can't find that function anywhere so it throws a reference error.